src/emailpro/account/emailpro-account.controller.js

```diff
@@ -169,5 +176,16 @@ angular.module('Module.emailpro.controllers').controller('EmailProTabAccountsCtr
     }
   };
 
+  $scope.addNewConfigureAccount = function () {
+    $scope.loadingNewConfiguredAccount = true;
+    EmailPro.getAccounts().then((accounts) => {
+      const newConfiguredAccount = _.find(accounts.list.results, account => /.*configureme\.me$/.test(account.domain));
```

Comment:
` /.*configureme\.me$/` could be a `constant` maybe ?

Reply:
Thanks @marie-j to point this out.

It could be handled in a separate tasks since this feature is actually in `To Prod` status.